discord_bots/commands.py

```diff
@@ -193,11 +163,11 @@ def get_even_teams(
 # Return n of the most even or least even teams
 # For best teams, use direction = 1, for worst teams use direction = -1
 def get_n_teams(
-    players: list[Player],
-    team_size: int,
-    is_rated: bool,
-    n: int,
-    direction: int = 1,
+        players: list[Player],
```

Comment:
nit: something weird is going on here. my vscode formats these back to 4 spaces. not a big deal but any chance you have an idea what the difference might be between our configs?